src/Microsoft.Framework.Runtime/NuGet/Authoring/ManifestReader.cs

```diff
@@ -50,6 +48,7 @@ private static ManifestMetadata ReadMetadata(XElement xElement)
                 {
                     ReadMetadataValue(manifestMetadata, element, allElements);
                 }
+
```

Comment:
Why can't this loop just be `foreach(var element in xElement.Elements())` ? Or can it?